src/transformers/integrations/integration_utils.py

```diff
@@ -812,7 +812,6 @@ def setup(self, args, state, model, **kwargs):
                         "initial_model": True,
                     },
                 )
-                model.save_pretrained(temp_dir)
```

Comment:
Could you update the comment above?

Reply:
@amyeroberts Just to check, is this directed at me or at @parambharat ?

Reply:
You :)

Reply:
Oh. You mean this one?
> # log the initial model and architecture to an artifact
Now that I read this again, I'm not completely sure anymore - maybe uploading the model checkpoint is intentional after all? If so, it should still respect the `WANDB_LOG_MODEL` setting, but maybe an extra `if` clause would be the better solution than removing this altogether in that case.
I'm really not sure though what the intent is - to me, uploading later (trained) LORA adapters makes much more sense than uploading the initial model, since the model you start from is usually stored in a permanent way elsewhere already anyway... 
For now I've updated the comment. 
@parambharat - could you weigh in on what the intent is here?

Reply:
I think the loading is intentional, but I agree that it might not be a good idea